### PR TITLE
#131 - update http

### LINKED ISSUE #131: Update http and use `BasicAuthSlice`
Let's update http and use `BasicAuthSlice` instead of `AuthSlice`.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #131 
Updated http and used `BasicAuthSlice`

### CODE REVIEW

File: src/main/java/com/artipie/pypi/http/PySlice.java [hunk 1]
```diff
@@ -61,17 +60,11 @@
...
-     * @param storage The storage and default parameters for free access.
-     * @param perms Access permissions.
-     * @param auth Auth details.
-     */
-    public PySlice(final Storage storage, final Permissions perms, final Authentication auth) {
-        this(storage, perms, new BasicIdentities(auth));
+        this(
+            storage,
+            Permissions.FREE,
+            (user, pswd) -> Optional.of(new Authentication.User("anonymous"))
```
Comment (olegmoz) [inline src/main/java/com/artipie/pypi/http/PySlice.java:66]: @olenagerasimova lets use newly added `Authentication.ANONYMOUS` here

File: src/test/java/com/artipie/pypi/http/PySliceITCase.java [hunk 2]
```diff
@@ -187,31 +186,24 @@ void canInstallWithVersion(@TempDir final Path temp) throws Exception {
...
         this.server = new VertxSliceServer(this.vertx, new PySlice(storage, perms, auth));
         final int port = this.server.start();
         Testcontainers.exposeHostPorts(port);
         return port;
     }
 
+    private int startServer(final Storage storage) {
+        return this.startServer(
+            storage, Permissions.FREE,
+            (name, pswd) -> Optional.of(new Authentication.User("anonymous"))
```
Comment (olegmoz) [inline src/test/java/com/artipie/pypi/http/PySliceITCase.java:205]: @olenagerasimova I'd suggest to release new version of `http` and use `Authentication.ANONYMOUS` here

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see two comments

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, approved

**Comment by 0crat** [comment]
Job `gh:artipie/pypi-adapter#132` is not assigned, can't get performer

**Pull request merged**